documentation/vespa-quick-start-multinode-aws.html

```diff
@@ -184,45 +143,54 @@ <h2 id="configure-vespa">Configure and deploy your Vespa application</h2>
   &lt;/content&gt;
 &lt;/services&gt;
 </pre>
-</li>
+<p>The <a href="reference/services.html"><samp>services.xml</samp></a> maps services to node aliases. 
+We have one <samp>configserver</samp>, two stateless jdisc search &amp; feed and two content nodes storing our data with a configured redundancy of 2.</p>
 
-<li><p><strong>Deploy the application :</strong></p>
+<p><strong>Deploy the application :</strong></p>
 <pre>
-[centos@ip-172-1-1-1 ~]$ vespa-deploy prepare src/main/application && vespa-deploy activate 
+vespa-deploy prepare src/main/application &amp;&amp; vespa-deploy activate 
```

Comment:
The config server was not started by the aws_bootstrap.sh script :/ I had to start it manually by `sudo service vespa-configserver start`.